Jenkinsfile

```diff
@@ -36,35 +37,82 @@ pipeline {
                             steps {
                                 sh 'docker version'
```

Comment:
it helps to find out which version and graph driver has been configured for the node.

unless your question is more along the lines of, "why does docker info have the server version info but not the client version info in the output?"

Reply:
Yes, the idea of printing these is to help with debugging (e.g. why does something fail on X, but pass on Y, which could be due to a different version of docker installed, or a different configuration.

Reply:
Ahh thanks for clearing that up! I was wondering if it was left here for debugging purposes.  It does make sense to have both since `docker info` doesn't show the client info